core/src/style/pointStyleBuilder.h

```diff
@@ -50,12 +55,20 @@ struct PointStyleBuilder : public StyleBuilder {
     bool addFeature(const Feature& _feat, const DrawRule& _rule) override;
 
 private:
+
+
+    Point interpolateSegment(const Line& _line, int i, int j, float distance);
```

Comment:
Those definitions can be removed, I think it's from the previous implementation before the line sampler was added.